src/signup.html

```diff
@@ -49,52 +49,47 @@
         <div class="signup">
             <div class="signup-screen">
                 <div class="app-title">
-
                     <h1> SignUp <img src="../pics/signup.png"></h1>
                 </div>
 
-
-
-                <form action='' method=" post">
+                <form id='signup'>
```

Comment:
why 2 form here

Reply:
In pink color was deleted, ok 
plz focus